### PR TITLE
Feat/from servicedesign to progettazioneservizi

### PR DESCRIPTION
**Pull request by danielaiozzo**
porting completed. 
index.rst not updated

### CODE REVIEW

File: doc/progettazione-di-servizi/analizzare-il-contesto-con-un-approccio-olistico-e-sistemico.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Prendendo in considerazione un servizio e tutti i soggetti coinvolti nella sua erogazione e fruizione (dall’utente finale all’operatore della Pubblica amministrazione)possiamo descrivere le loro relazioni evidenziando i passaggi di informazioni, documenti, denaro o altro valore, che intercorrono tra l’uno e l’altro. Le mappe dell’ecosistema vengono costruite **mettendo al centro il cittadino e disponendo attorno a lui tutti i soggetti interessati**: più vicino quelli maggiormente a contatto con l’utente e mano a mano più lontano quelli con le relazioni più deboli o nascoste. In un secondo momento vengono tracciate delle **linee di collegamento che forniscono l’informazione relativa allo scambio che avviene tra ciascun soggetto e soggetti vicini, costruendo un’immagine completa della struttura su cui si basa il servizio**.
```
Comment (bfabio) [inline doc/progettazione-di-servizi/analizzare-il-contesto-con-un-approccio-olistico-e-sistemico.rst:28]:
```suggestion
Prendendo in considerazione un servizio e tutti i soggetti coinvolti nella sua erogazione e fruizione (dall’utente finale all’operatore della Pubblica amministrazione) possiamo descrivere le loro relazioni evidenziando i passaggi di informazioni, documenti, denaro o altro valore, che intercorrono tra l’uno e l’altro. Le mappe dell’ecosistema vengono costruite **mettendo al centro il cittadino e disponendo attorno a lui tutti i soggetti interessati**: più vicino quelli maggiormente a contatto con l’utente e mano a mano più lontano quelli con le relazioni più deboli o nascoste. In un secondo momento vengono tracciate delle **linee di collegamento che forniscono l’informazione relativa allo scambio che avviene tra ciascun soggetto e soggetti vicini, costruendo un’immagine completa della struttura su cui si basa il servizio**. 
```

File: doc/progettazione-di-servizi.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Il concetto di servizio esiste da tempo, quello che è emerso da più recente è la necessità e l’opportunità di applicare delle metodologie di progettazione per definire come dovrebbe avvenire la relazione tra utenti e fornitori del servizio stesso, affinché si generino esperienze di qualità e valore per tutte le parti coinvolte. In contrapposizione alla rivoluzione industriale, la rivoluzione dei servizi è sorta dalla necessità di far fronte a nuove forme di differenziazione da parte dei produttori di beni, così che l’associazione di prodotti e servizi ha generato valore economico basato sull’erogazione di esperienze e non solo di prodotti. 
+
+L’evoluzione tecnologica, l’evoluzione dei sistemi informatici, la rivoluzione di internet dagli anni 2000 hanno contribuito alla diffusione dei servizi digitali. L’unione tra prodotti e servizi, rafforzati dalle nuove tecnologie hanno portato al fenomeno della servitization (il processo per cui una azienda/ente che produce e vende beni o prodotti trasforma questi ultimi in una vendita di servizi), inducendo una crescita esponenziale del ruolo che i servizi hanno per la nostra vita, anche in ambito pubblico.
```
Comment (bfabio) [inline doc/progettazione-di-servizi.rst:16]:
```suggestion
L’evoluzione tecnologica, l’evoluzione dei sistemi informatici, la rivoluzione di Internet dagli anni 2000 hanno contribuito alla diffusione dei servizi digitali. L’unione tra prodotti e servizi, rafforzati dalle nuove tecnologie hanno portato al fenomeno della servitization (il processo per cui una azienda/ente che produce e vende beni o prodotti trasforma questi ultimi in una vendita di servizi), inducendo una crescita esponenziale del ruolo che i servizi hanno per la nostra vita, anche in ambito pubblico.
```

File: doc/progettazione-di-servizi.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+I servizi sono per definizione immateriali, ovvero si esauriscono nel momento in cui vengono fruiti, possono essere usati ma non posseduti, non sono fisici né trasportabili. I *touchpoint* sono il mezzo attraverso il quale si materializza un servizio. Il *service design* studia e progetta il modo in cui gli utenti usufruiscono dei servizi tramite i touchpoint fisici o digitali di cui si avvale chi offre il servizio. Tra i *touchpoint* troviamo per esempio canali di interazione diretta tra utenti e organizzazione: un ufficio relazioni con il pubblico, un call center, una chat con un operatore, un'applicazione, un sito internet, la pagina di un social network. Il servizio digitale erogato tramite il touchpoint deve essere **di facile utilizzo, eventualmente corredato da un contesto di informazioni sintetiche e chiare, deve essere accessibile e disponibile a tutta la collettività**.
```
Comment (bfabio) [inline doc/progettazione-di-servizi.rst:20]:
```suggestion
I servizi sono per definizione immateriali, ovvero si esauriscono nel momento in cui vengono fruiti, possono essere usati ma non posseduti, non sono fisici né trasportabili. I *touchpoint* sono il mezzo attraverso il quale si materializza un servizio. Il *service design* studia e progetta il modo in cui gli utenti usufruiscono dei servizi tramite i touchpoint fisici o digitali di cui si avvale chi offre il servizio. Tra i *touchpoint* troviamo per esempio canali di interazione diretta tra utenti e organizzazione: un ufficio relazioni con il pubblico, un call center, una chat con un operatore, un'applicazione, un sito Internet, la pagina di un social network. Il servizio digitale erogato tramite il touchpoint deve essere **di facile utilizzo, eventualmente corredato da un contesto di informazioni sintetiche e chiare, deve essere accessibile e disponibile a tutta la collettività**.
```

File: doc/progettazione-di-servizi.rst [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+Nell’ambito della trasformazione digitale il service design si focalizza sull’esperienza concernente l’interazione tra l’utente e il *touchpoint* digitale, il primo può essere direttamente il cittadino, oppure un operatore della Pubblica Amministrazione.
+
+La progettazione di servizi al cittadino
+----------------------------------------
+La Pubblica Amministrazione gestisce e fornisce un grande numero di servizi al cittadino, o servizi di interesse pubblico. Oggi buona parte dei servizi forniti dalla Pubblica Amministrazione esiste sia in forma analogica (es. uffici di relazione con il pubblico, sportelli), che in forma digitale (es. sito web del comune, pagina social di un Ministero). Le nuove possibilità offerte dal digitale hanno portato le pubbliche amministrazioni che erogano servizi a dover rivedere il modo in cui costruiscono e gestiscono la relazione con i propri cittadini.
```
Comment (bfabio) [inline doc/progettazione-di-servizi.rst:26]:
```suggestion
La Pubblica Amministrazione gestisce e fornisce un grande numero di servizi al cittadino, o servizi di interesse pubblico. Oggi buona parte dei servizi forniti dalla Pubblica Amministrazione esiste sia in forma analogica (es. uffici di relazione con il pubblico, sportelli), che in forma digitale (es. sito web del Comune, pagina social di un Ministero). Le nuove possibilità offerte dal digitale hanno portato le pubbliche amministrazioni che erogano servizi a dover rivedere il modo in cui costruiscono e gestiscono la relazione con i propri cittadini. 
```

File: doc/progettazione-di-servizi/progettare-un-servizio-pubblico-digitale.rst [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+- si basa sulla ricerca, mette al centro delle sue indagini i cittadini e gli operatori pubblici, creando scenari d’uso e profili utili a comprenderne bisogni e necessità;
+- include nella progettazione tutte le persone che concorrono all’erogazione, fruizione e comunicazione  del servizio stesso, dai cittadini agli operatori, ai decisori (*decision makers*);
+- prevede un approccio olistico alla risoluzione del problema, considerando l'intero ecosistema del servizio;
+- procede per ipotesi, sviluppa prototipi e raffina il prodotto/servizio con test di usabilità con gli utenti finali, siano essi cittadini o operatori della Pubblica Amministrazione;
+- data la natura immateriale e l’intangibilità del servizio, la sua progettazione si avvale di strumenti per visualizzare e analizzare la complessità.
+
+Gli elementi da considerare nella progettazione di un servizio sono:
+
+- **stakeholder** (portatori di interesse): coinvolgere individui e/o le organizzazioni che influenzano l’esperienza d’uso di un servizio, contribuiscono alla sua erogazione , e fruizione;
```
Comment (bfabio) [inline doc/progettazione-di-servizi/progettare-un-servizio-pubblico-digitale.rst:17]:
```suggestion
- **stakeholder** (portatori di interesse): coinvolgere individui e/o le organizzazioni che influenzano l’esperienza d’uso di un servizio, contribuiscono alla sua erogazione e fruizione;
```

**Comment by danielaiozzo** [comment]
done @bfabio

**Comment by bfabio** [comment]
We need to update `index.rst`, right?

**Comment by danielaiozzo** [comment]
No, I will update the index file after your merge

Il Mer 2 Feb 2022, 13:36 Fabio Bonelli ***@***.***> ha
scritto:

**Review by bfabio — APPROVED** [review]

**Pull request merged**